api/views/yangSearch/yangSearch.py

```diff
@@ -569,6 +571,28 @@ def get_modules_revision_organization(module_name: str, revision: t.Optional[str
         abort(404, f'Failed to get revisions and organization for {name_rev}')
 
 
+def get_modules_revision_maturity_levels(module_name: str) -> list[dict[str, str]]:
```

Comment:
This is very similar to `get_modules_revision_organization()`. Instead of adding a new function, `get_modules_revision_organization()` could return this `revision: rfc` dict instead of the list of revision.

Reply:
yeah, you are right, I'll change that